usecases/modulecomponents/batch/batch_test.go

```diff
@@ -176,8 +182,8 @@ func TestBatchTimeouts(t *testing.T) {
 		{batchTime: 1 * time.Second, expectedErrors: 0},
 	}
 	for _, tt := range cases {
-		t.Run("", func(t *testing.T) {
-			v := NewBatchVectorizer(client, tt.batchTime, 2000, 2000.0, logger) // avoid waiting for rate limit
+		t.Run("BatchTimeouts", func(t *testing.T) {
```

Comment:
I mean there's a list of `cases` that you are testing and now each case will be run with the same name, can we make it like this `t.Run(tt.name, ...)`?